README.md

```diff
@@ -214,7 +214,7 @@ The folder structure should look like this:
 ```
 .
 └── my-pm-data-server
-    └── my-graph
+    └── graph_data
```

Comment:
should the reference below:

```
Update the config.js in this repository to point to your data server, and your graph should be accessible at

http://127.0.0.1:8081/#/galaxy/my-graph
```

Also be changed to `graph_data`  ?

Reply:
Please close this pull request. I am not sure anymore why https://github.com/anvaka/pm/issues/24 mentions needing to change `my-graph` to `graph_data` since I was able to make `my-graph` work. I added my steps, so hopefully that will help.